Adafruit_Si7021.cpp

```diff
@@ -131,43 +196,68 @@ void Adafruit_Si7021::readSerialNumber(void) {
   sernum_b <<= 8;
   sernum_b |= Wire.read();
   Wire.read();
+
+    switch(sernum_b >> 24) {
+        case 0:
+        case 0xff:
+            _model = "SI engineering samples";
```

Comment:
chars are ugly, prefer `enum`
chars are more difficult to handle in a program
chars are present in RAM which are a real thread on AVR arduinos
this lines of code will eat about `65 byte` of RAM!